datagrepper/default_config.py

```diff
@@ -3,5 +3,5 @@
 CORS_METHODS = ["GET", "OPTIONS"]
 CORS_HEADERS = [".*"]
 CORS_MAX_AGE = "600"
-DEFAULT_QUERY_DELTA = None
+DEFAULT_QUERY_DELTA = 0
```

Comment:
Do you prefer we set a default here or we set it in the config file in Ansible?

Reply:
i'm happy to leave this as it was -- and just set our appropriate default delta in our deployment config

Reply:
Okay